lib/ssl/src/ssl_connection.hrl

```diff
@@ -93,7 +93,8 @@
 -record(connection_env, { 
                           user_application      :: {Monitor::reference(), User::pid()},
                           downgrade,
```

Comment:
Why not have a type spec for the downgrade field as well. Should be `{Pid::pid(),From:pid()}` i think. The question is what to name the first element with a better name than Pid.

Reply:
I made it {NewController::pid(), Caller::gen_statem:from()}